package.json

```diff
@@ -33,8 +33,8 @@
     "solhint": "^2.1.0",
     "truffle": "^5.0.32",
     "ganache-time-traveler": "^1.0.5",
-    "truffle-flatten": "^1.0.5",
-    "truffle-verify": "^1.0.3",
+    "truffle-flatten": "^1.0.6",
+    "truffle-verify": "^1.0.4",
```

Comment:
shall we remove hats? to fix the version and increase it manually?

Reply:
can do

Reply:
i don't think so, did the upgrade for visibility but based on tests yarn will upgrade if needed.